src/applications/ivc-champva/10-10D/pages/ApplicantSponsorMarriageDetailsPage.jsx

```diff
@@ -0,0 +1,84 @@
+import ApplicantRelationshipPage, {
```

Comment:
Essentially just a wrapper around `ApplicantRelationshipPage`, with a function to generate our custom-worded radio options that get fed into that component.